tests/stress/benchmarkstress_test.go

```diff
@@ -0,0 +1,90 @@
+package system
+
+import (
+	"strconv"
+	"testing"
+)
+
+const (
+	deployCommand = "while true ; do echo LONG RUNNING SERVICE ; sleep 5 ; done"
+)
+
+func BenchmarkStress25Environments0Deploys0Services0LoadBalancers(b *testing.B) {
```

Comment:
Seems like there's a few different approaches you could take here that would make these function names not have to be oddly named. However, I'm guessing you did it this way because then each benchmark gets it's own context/duration in the resulting output? 

Open question then becomes: what's more important? Readable output of our benchmarks or readable code? I think I personally prefer the latter.

Reply:
The 1-function-per-case for benchmark testing is a standard Go thing (see: https://dave.cheney.net/2013/06/30/how-to-write-benchmarks-in-go). So I think the pattern is correct, just the names of function themselves could be improved.

Reply:
Will adjust names as suggested below.

✅